src/components/icon/icons/lightning.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function Lightning({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest} viewBox={'0 0 13 19'}>
```

Comment:
* Replace `LightningBolt` icon with this instead?
* Move `viewBox` to defaultProps
* Combine the `g` tags
* Remove `stroke` from propTypes and from the arguments list since it is not used